packages/styled-base/src/index.js

```diff
@@ -221,7 +217,7 @@ let createStyled: CreateStyled = (tag: any, options?: StyledOptions) => {
       )(...styles)
     }
     return Styled
-  }
+  }: CreateStyledComponent)
```

Comment:
is this extra cast needed? Styled is being returned already so it should be an indicator of the containing function's type. Maybe it's just me but this looks a little bit strange

Reply:
No I just added it for clarity, I can remove it